json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -456,11 +456,11 @@ public function update_settings() {
 
 				case Jetpack_SEO_Utils::FRONT_PAGE_META_OPTION:
 					if ( ! Jetpack_SEO_Utils::is_enabled_jetpack_seo() && ! Jetpack_SEO_Utils::has_grandfathered_front_page_meta() ) {
-						return new WP_Error( 'unauthorized', __( 'SEO tools are not enabled for this site.', 'Jetpack' ), 403 );
+						return new WP_Error( 'unauthorized', __( 'SEO tools are not enabled for this site.', 'jetpack' ), 403 );
```

Comment:
All these `__` should be `esc_html__` since it's safe that way. Same applies to `_x`, should be `esc_html_x`